### ISSUE TITLE
Transition to Login View after selection of lms item does not take place properly

### ISSUE THREAD

**Issue by evangelia-mitsopoulou** [ISN-Zurich/ISN-Learning-Cards#151]
Current Behavior: When we select an lms list item for the very first time (they have not been registered) the transition to login view takes place as normal. When the select an LMS again, the transition to  login view doesn't take place. We can only see the selection of the new lms only after we click on the close button.

Desired Behavior: When select an LMS to ALWAYS be forwarded to the login view.

HINT: This is happening because the controller handles the transition to login view only when the registration has been set. We should add in the clickLMSItem()  the transition to takes place if we already have a client key.

**Comment by evangelia-mitsopoulou** [comment]
Invalid the above. The problem exists only when click on the Pfp LMS. we create a new issue for this
